R/get_baseline_predictions.R

```diff
@@ -0,0 +1,136 @@
+#' Get predictions for a single baseline model
+#'
+#' @param target_ts a `data.frame` of target data in a time series format
+#'   (contains columns `time_index`, `location`, and `observation`) for a single
+#'   location
+#' @param transformation string specifying the transformation used on the
+#'   distribution which determines its shape; can be one of "none" or "sqrt".
+#' @param symmetrize boolean specifying whether to make the distribution symmetric;
+#'   can be one of `TRUE` or `FALSE`.
+#' @param window_size integer specifying how many previous observations in the
+#'   target data should be used to inform the forecasts
+#' @param effective_horizons numeric vector of prediction horizons relative to
+#'   the last observed date in `target_ts`
+#' @param origin string specifying the origin to use when making predictions;
+#'   recommended to be "median" if the temporal resolution is daily and "obs"
+#'   if weekly or otherwise. Defaults to "obs".
+#' @param quantile_levels numeric vector of quantile levels to output; set to NULL
+#'   if quantile outputs not requested. Defaults to NULL.
+#' @param n_samples integer of amount of samples to output (and predict);
+#'   set to NULL if sample outputs not requested (in this case 100000 samples
+#'   are generated from which to extract quantiles). Defaults to NULL.
```

Comment:
I see from code below that the answer to 1 is "yes" -- if we decide to keep this functionality, it would be good to clarify that in the documentation here because I wasn't sure of that just from reading the docs.

Reply:
Decided to split `n_samples` into two arguments: `n_sim` for the number of simulations and `n_samples` for the number of samples to actually return